contracts/reverseRegistrar/profiles/L2ReverseResolverBase.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.4;
+
+import "@openzeppelin/contracts/utils/introspection/ERC165.sol";
+import "../../resolvers/profiles/IVersionableResolver.sol";
+
+abstract contract L2ReverseResolverBase is ERC165 {
+    mapping(bytes32 => uint64) internal recordVersions;
+    event VersionChanged(bytes32 indexed node, uint64 newVersion);
+    bytes32 public immutable L2_REVERSE_NODE;
+
+    bytes32 constant lookup =
```

Comment:
Can you add a comment explaining what `lookup` is?

Reply:
I'm actually not 100% sure, this is lifted from the original reverse registar

Reply:
@Arachnid do you know what this is?

Reply:
It's just '0123456789abcdef' in hex form; it allows quickly mapping from a number from 0-15 to a hexadecimal character.